### PR TITLE
Remove sendEmail method

### PR DESCRIPTION
**Pull request by zhx828**
Use sentry to capture exception instead.

This fixes https://github.com/oncokb/oncokb/issues/3471

### CODE REVIEW

**Comment by zhx828** [comment]
@calvinlu3 could you do me a favor double check this pr looks good? Thank you.

File: app/scripts/controllers/emailDialogCtrl.js
Comment (calvinlu3) [inline app/scripts/controllers/emailDialogCtrl.js:1]: In `index.html`, we can remove this script on line 202

File: app/scripts/app.spec.js
Comment (calvinlu3) [inline app/scripts/app.spec.js:1]: We don't need this test anymore? In the README, should probably update the **Testing** section

**Review by calvinlu3 — APPROVED** [review]

File: app/scripts/app.spec.js
Comment (zhx828) [reply app/scripts/app.spec.js:1]: The whole testing is not running at the moment... I will delete the whole testing section.

**Pull request merged**